### PR TITLE
Generate Patches for Go 1.18

### PR DESCRIPTION
**Pull request by zafs23**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/golang/go/1.18/rpmbuild/SPECS/golang.spec [hunk 1]
```diff
@@ -164,6 +164,10 @@ Patch101:       0101-syscall-expose-IfInfomsg.X__ifi_pad-on-s390x.patch
 Patch102:       0102-cmd-go-disable-Google-s-proxy-and-sumdb.patch
 Patch103:       0103-Don-t-use-the-bundled-tzdata-at-runtime-except-for-t.patch
 Patch104:       0104-add-method-to-skip-privd-tests-if-required.patch
+Patch105:       0105-update-bundled-golang.org.patch
```
Comment (markapruett) [inline projects/golang/go/1.18/rpmbuild/SPECS/golang.spec:167]: Aren't the 100 series patches for our own patches to golang, and not the backported ones?

File: projects/golang/go/1.18/rpmbuild/SPECS/golang.spec [hunk 2]
```diff
@@ -160,10 +160,19 @@ Requires:       %{name}-bin = %{version}-%{release}
...
+Patch4:       0004-go-1.18.10-path-filepath-do-not-Clean-relative-path..patch
+
 Patch101:       0101-syscall-expose-IfInfomsg.X__ifi_pad-on-s390x.patch
 Patch102:       0102-cmd-go-disable-Google-s-proxy-and-sumdb.patch
 Patch103:       0103-Don-t-use-the-bundled-tzdata-at-runtime-except-for-t.patch
 Patch104:       0104-add-method-to-skip-privd-tests-if-required.patch
+Patch105:       0105-update-bundled-golang.org.patch
+Patch106:       0106-replace-all-usages-of-BytesOrPanic.patch
+Patch107:       0107-multipart-limit-memory-inode-consumption.patch
+Patch108:       0108-path-filepath-do-not-Clean-relative-path..patch
```
Comment (rcrozean) [inline projects/golang/go/1.18/rpmbuild/SPECS/golang.spec:175]: Now that you fixed the naming of the patches, you need to remove these files from the spec.

**Pull request merged**